lib/prop/limiter.rb

```diff
@@ -46,8 +46,7 @@ def before_throttle(&blk)
       #
       # Raises Prop::RateLimited if the number if the threshold for this handle has been reached
       def configure(handle, defaults)
-        raise ArgumentError.new("Invalid threshold setting") unless defaults[:threshold].to_i > 0
-        raise ArgumentError.new("Invalid interval setting")  unless defaults[:interval].to_i > 0
+        Prop::IntervalStrategy.validate_options!(defaults)
```

Comment:
I guess this should not reference `Prop::IntervalStrategy`, as other strategies also use `configure`. Something like:
```
       Prop::Options.validate_options!(defaults)
```
With `lib/prop/options.rb`:
```
   def self.validate_options!(options)
     # strategy = work out strategy from options
     strategy.validate_options!(options)
   end
```